meta-lmp-support/recipes-bsp/u-boot/u-boot-fio/uz/0001-Read-MAC-from-EEPROM-on-Xilinx.patch

```diff
@@ -0,0 +1,42 @@
+From 639eb84dad63d917b325b25fabefebb4f610e3ec Mon Sep 17 00:00:00 2001
+From: Ari Parkkila <ari.parkkila@pelion.com>
+Date: Wed, 24 Feb 2021 15:50:45 +0200
+Subject: [PATCH] Read MAC from EEPROM on Xilinx
+
+
+diff --git a/meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/device-tree/device-tree/uz3eg-iocc/system-board.dtsi b/meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/device-tree/device-tree/uz3eg-iocc/system-board.dtsi
+index 99eced2..353a111 100644
+--- a/meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/device-tree/device-tree/uz3eg-iocc/system-board.dtsi
++++ b/meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/device-tree/device-tree/uz3eg-iocc/system-board.dtsi
+@@ -30,6 +30,11 @@
+ 			linux,default-trigger = "heartbeat";
+ 		};
+ 	};
++
++        chosen {
++                xlnx,eeprom = &eeprom; /* EMAC on EEPROM */
++        };
```

Comment:
Mixed spaces with tabs in the original

Reply:
To minimize changes, this patch uses spaces without fixing original file `meta-lmp/meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/device-tree/device-tree/uz3eg-iocc/system-board.dtsi` (which mixes spaces and tabs).